lib/pbench/server/indexer.py

```diff
@@ -539,22 +539,34 @@ def _get_es_hosts(config, logger):
     Return list of dicts (a single dict for now) - that's what ES is expecting.
     """
     try:
-        URL = config.get("Indexing", "server")
+        host = config.get("elasticsearch", "host")
+        port = config.get("elasticsearch", "port")
     except NoSectionError:
         logger.warning(
-            "Need an [Indexing] section with host and port defined"
-            " in {} configuration file",
+            "Do not have [elasticsearch] section with host and port defined"
+            " in {} configuration file, checking the Indexing section",
             " ".join(config.files),
         )
-        return None
+        try:
+            URL = config.get("Indexing", "server")
```

Comment:
Seems like we can just drop the reference to the `server` option of the `Indexing` section entirely.